obelisk/cpp/obelisk_cpp/include/obelisk_mujoco_sim_robot.h

```diff
@@ -0,0 +1,630 @@
+#pragma once
+
+#include <cstdarg>
+#include <filesystem>
+
+#include <GLFW/glfw3.h>
+#include <mujoco/mujoco.h>
+
+#include "obelisk_sensor_msgs/msg/joint_encoders.hpp"
+#include "obelisk_sim_robot.h"
+
+// TODO: Fix issue related to ENV XDG_RUNTIME_DIR=/run/user/1000  # Replace 1000 with your user ID
```

Comment:
we can probably set this in the dev container stuff/setup script

Reply:
added